compiler/resolution/functionResolution.cpp

```diff
@@ -6082,6 +6082,25 @@ resolveExpr(Expr* expr) {
           // delayed instance to have the right type.
           actualSym->type = formal->type;
           actualSym->removeFlag(FLAG_DELAY_GENERIC_EXPANSION);
+
```

Comment:
Hmmm.  Further consideration of call->numActuals() leads me to be more confused than I was
before.  I am going to keep working on this but I suspect we'll want to thrash this out face to face
rather than by email